docs/tips/16-references-to-components.md

```diff
@@ -25,4 +25,4 @@ var myComponentInstance = React.render(myComponentElement, myContainer);
 
 > Note:
 >
-> This should only ever be used at the top level. Inside components, let your `props` and `state` handle communication with child components, and only reference components via [refs](http://facebook.github.io/react/docs/more-about-refs.html).
+> This should only ever be used at the top level. Inside components, let your `props` and `state` handle communication with child components, and only reference components via [refs](https://facebook.github.io/react/docs/more-about-refs.html).
```

Comment:
We should probably just make this a relative link (which is what we typically do, but can follow up later)